EIPS/eip-7851.md

```diff
@@ -0,0 +1,247 @@
+---
+eip: 7851
+title: EOA private key deactivation/reactivation
```

Comment:
```suggestion
title: Delegated EOA's private key deactivation/reactivation
```
may be doesn't fit the title length constraint...

Reply:
yes. While the added `delegated` seems necessary, what about `deactivate/reactivate of delegated EOA's key`?

Reply:
yes sounds good